src/main/java/com/Elessar/app/server/LogOffHandler.java

```diff
@@ -42,7 +44,9 @@ public void handle(HttpExchange he) throws IOException {
             final LogoffResponse.Builder logoffResponse = LogoffResponse.newBuilder();
             final String userName = logoffRequest.getName();
 
+            monitor.timerStart(MyDatabase.UPDATE);
```

Comment:
A better place might be log the metric inside db.update() function so we get the metrics whenever the method is called.